hazelcast-client/src/test/resources/hazelcast-cache-entrylistener-test.xml

```diff
@@ -20,9 +20,10 @@
     Please see the schema for how to configure Hazelcast at https://hazelcast.com/schema/config/hazelcast-config-3.10.xsd
     or the documentation at https://hazelcast.org/documentation/
 -->
-<hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.10.xsd"
-           xmlns="http://www.hazelcast.com/schema/config"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<hazelcast xmlns="http://www.hazelcast.com/schema/config"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.hazelcast.com/schema/config
+           http://www.hazelcast.com/schema/config/hazelcast-config-3.10.xsd">
```

Comment:
Can we switch to versionless URI here? Same as on the member XMLs, applies for all XMLs.

Reply:
I don't think there is a `spring.schemas` concept for normal XML files (or my Google search skills are to bad). At least it's not implemented, so there is no place to add such a mapping.